DigitalLearningSolutions.Web/Controllers/ForgotPasswordController.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace DigitalLearningSolutions.Web.Controllers
+{
+    using Microsoft.AspNetCore.Mvc;
+    using DigitalLearningSolutions.Web.ViewModels.ForgotPassword;
+    using DigitalLearningSolutions.Web.ViewModels.LearningSolutions;
+
+    public class ForgotPasswordController : Controller
+    {
+
+        [Route("/ForgotPassword")]
+        public IActionResult ForgotPassword()
+        {
+            var model = new ForgotPasswordViewModel();
+            return View(model);
+        }
+
+        [HttpPost]
+        public IActionResult SubmitEmail(string emailAddress)
+        {
+            //Do some validation
+            ConfirmationViewModel confirmationModel = GetForgotPasswordConfirmationViewModel();
+
+            return View("~/Views/LearningSolutions/Confirmation.cshtml", confirmationModel);
+        }
+
+        private ConfirmationViewModel GetForgotPasswordConfirmationViewModel()
+        {
+            string pageTitle = "Password Reset Email Sent";
+            string[] pageDescription =
+            {
+                "An email has been sent to you giving details of how to reset your password.",
+                "The link provided in that email will expire in two hours.",
+                "If you have not received an email, please contact your centre administrator."
+            };
+
+            return new ConfirmationViewModel(pageTitle, pageDescription);
```

Comment:
Hmm.. My initial thought was that we’ll have lots of confirmation pages with just a title and description that might be worth commonising. However, given that there are several paragraphs that we might need to add here and there, pulling the content away from the HTML files doesn’t seem to be worth the commonisation. What are your thoughts? Would you be happy to change this to just forgot password specific confirmation page and reverting the commonisation?

Reply:
I am happy to revert the commonisation and just use regular HTML files for this and future confirmation pages. I think that the impact will be very minor either way. Probably depends on how many confirmation pages we think we will need, how complex they will be, and how often they will need modifying, but I don't currently have a wide understanding across the whole project of these factors. Do we anticipate lots of confirmations that aren't just a couple of lines?

Reply:
Yeah, I think the registration will have a much more complex confirmation page and sign in page won't need one. We might need one when the password has been reset successfully, but I don't think there's too much of it. Maybe we should revert for now and reconsider later if we feel like there's a lot of repetition across the project? :)